layouts/_partials/components/analytics/goat-counter.html

```diff
@@ -0,0 +1,17 @@
+{{- with .Site.Params.analytics.goatCounter -}}
+    {{- if .code -}}
+        <script 
+            data-goatcounter="https://{{- .code -}}.goatcounter.com/count"
+            data-goatcounter-settings='
+            {
+                "no_onload": {{- if .noOnload }}true{{- else }}false{{- end -}},
+                "no_events": {{- if .noEvents }}true{{- else }}false{{- end -}},
+                "allow_local": {{- if .allowLocal }}true{{- else }}false{{- end -}},
+                "allow_frame": {{- if .allowFrame }}true{{- else }}false{{- end -}}
```

Comment:
```suggestion
                "no_onload":{{ .onLoad | default false }},
                "no_events":{{ .noEvents | default false }},
                "allow_local":{{ .allowLocal | default false }},
                "allow_frame":{{ .allowFrame | default false }}
```
You can simplify this part.